frontend/src/components/Form/TextInput/textinput.cy.tsx

```diff
@@ -0,0 +1,122 @@
+import { Form, Formik } from "formik";
+import 'material-symbols';
+import * as yup from 'yup';
+import '../../../styles/global.css';
+import { TextInput } from "./TextInput";
+
+const initialValues = {
+    title: ""
+}
+
+const submitForm = (values: typeof initialValues) => {
+    console.log(values);
+}
+
+const schema = yup.object().shape({
+    title: yup.string().required("Title is required"),
+})
+
+const Wrapper: React.FC<{ children: React.ReactNode }> = ({ children }) => {
+    return <Formik initialValues={initialValues} onSubmit={submitForm} validationSchema={schema} >
+        <Form>
+            <div className="w-80">
+                {children}
+            </div>
+        </Form>
+    </Formik>
+}
+
+const data = {
+    name: "title",
+    placeholder: "Hello...",
+    labelText: "Titel",
+    helperText: "The title is shown at the top of the page.",
+    type: "email"
+}
+
+describe('Text Input', () => {
+    describe('Basic Props', () => {
+        beforeEach(() => {
+            cy.mount(<Wrapper>
+                <TextInput name={data.name} placeholder={data.placeholder} labelText={data.labelText} helperText={data.helperText} />
+            </Wrapper>)
+        })
+
+        it('should set name in input attribute', () => {
+            cy.get('input').invoke('attr', 'name').should('contain', data.name)
```

Comment:
Am I mistaken or does this just test your `beforeEach` function and not the actual component?